ui/components-react/src/components-react/newEditors/values/Metadata.ts

```diff
@@ -0,0 +1,41 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+/**
+ * Value types supported by editor system.
+ * @beta
+ */
+export type ValueType =
+  | "string"
+  | "number"
+  | "bool"
+  | "date"
+  | "enum"
+  | "instanceKey";
+
+/**
+ * Additional metadata that is used along side value to determine applicable editor.
+ * @beta
+ */
+export interface ValueMetadata {
```

Comment:
maybe this should also have information if the value is nullable? in our case most properties will be nullable, but some won't (e.g. some navigation properties like `BisCore:Element.Model`)

Reply:
Added `isNullable`